disco/voice/playable.py

```diff
@@ -163,7 +163,7 @@ def info(self):
                 if self._url:
                     obj = ydl.extract_info(self._url, download=False, process=False)
                     if 'entries' in obj:
-                        self._ie_info = obj['entries'][0]
+                        self._ie_info = list(obj['entries'])[0]
```

Comment:
This is a generator in Py3 or?

Reply:
Yes

`obj['entries'] = <generator object YoutubeEntryListBaseInfoExtractor._entries at 0x...>`